testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -387,23 +383,20 @@ impl TxEmitter {
                     e
                 )
             })?[0];
-        Ok(AccountData {
-            address,
-            key_pair: mint_key_pair.clone(),
-            sequence_number,
-        })
+        Ok(LocalAccount::new(address, account_key, sequence_number))
     }
 
     pub async fn get_money_source(
         &self,
         instances: &[Instance],
         coins_total: u64,
-    ) -> Result<AccountData> {
+    ) -> Result<LocalAccount> {
         let client = self.pick_mint_instance(instances).json_rpc_client();
         let faucet_account = if !self.vasp {
             info!("Creating and minting faucet account");
             let mut account = self.load_faucet_account(&client).await?;
-            let mint_txn = gen_mint_request(&mut account, coins_total, self.chain_id);
+            let tx_factory = TransactionFactory::new(self.chain_id);
```

Comment:
Can we have the factory stored inside of the TxEmitter struct so that these methods can reference it instead of needing to construct one each time?